apps/web/modules/survey/hooks/useSingleUseId.tsx

```
@@ -12,13 +12,15 @@ export const useSingleUseId = (survey: TSurvey | TSurveyList) => {
     if (survey.singleUse?.enabled) {
-      const response = await generateSingleUseIdAction({
+      const response = await generateSingleUseIdsAction({
         surveyId: survey.id,
         isEncrypted: !!survey.singleUse?.isEncrypted,
+        count: 1,
       });
-      if (response?.data) {
-        setSingleUseId(response.data);
-        return response.data;
+
+      if (!!response?.data?.length) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)